docs/claimantmodel/Book.md

```diff
@@ -0,0 +1,130 @@
+While this allows those familiar with CT to reach consensus on a rough design quickly, it has a number of drawbacks:
+
+1. Those that don't already understand CT now have to read [RFC 6962](https://www.rfc-editor.org/rfc/rfc6962) to understand
+   the domain that the other designers are drawing from
+1. There are still terms in CT that people disagree on the precise definition of (e.g. Auditor and Monitor)
+1. CT uses mechanisms such as SCTs that transparency systems should aim to avoid unless strictly necessary
+1. Any differences in the roles, interactions, actors, or motivations between CT and the new system being designed can
+   have knock-on effects that can undermine the security properties achieved
+
+With this in mind, a clean, role-based model for describing transparency systems was required, and the Claimant Model was born.
+
+## Applicability
+
+The Claimant Model can be used to describe any system where an actor (`Believer`) performs some trusted action based on some
+information (`Claim`) provided to them by a trusted third-party (`Claimant`). There is _usually_ an asymmetry, where the
+Believer can not verify that the Claim is true, but believes it because they trust the Claimant.
+
+The situation above appears in countless settings in our everyday human lives.
+On a technical level, this also happens frequently, often invisibly to the user.
+For example, when a Chrome browser (`Believer`) receives a TLS certificate (`Claim`) for a domain, it only performs
+the connection (trusted action) if the certificate is issued by a Certificate Authority (`Claimant`) that chains to its
+trusted roots.
+As another example, a software package manager (`Believer`) will only install software (trusted action) when the package
+(`Claim`) is signed by the correct package maintainer (`Claimant`).
+
+Above we state that it must be possible to verify that the Claim is true.
+This means that all Claims must be _falsifiable_, i.e. they are able to be proven false.
+We'll discuss _who_ can falsify Claims later.
+
+### Exceptions
+
+There are some situations that the Claimant Model _could_ be used to describe, but taking the time to do so would be a poor
+return on investment, or more simply, just confuse a situation that can be easily stated in simpler terms.
+
+- Believer can trivially verify the Claim: trust in the Claimant is a non-factor in this case. For example, if a web
+  service exists that can return the prime factors of some large number, the Believer may not have been able to perform
+  this factorization itself, but it can trivially multiply the factors together to confirm the result.
+- Believer will trust Claims from anyone: if the identity of the Claimant is not a factor in the Believer's decision to
+  perform some action, then the Claimant Model provides little benefit. Such situtations are unfortunately
+  still common, e.g. installing unsigned software/firmware.
```

Comment:
Either in this section or somewhere, is it worth explicitly stating that this expects that claims are present/discoverable in a transparency log? That's something that wasn't immediately obvious to me learning the model.